src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -83,112 +56,13 @@
-    <Compile Include="Linker\MonoDroid.Tuner\PreserveHttpAndroidClientHandler.cs" />
-    <Compile Include="Linker\MonoDroid.Tuner\StripEmbeddedLibraries.cs" />
-    <Compile Include="Tasks\Aapt.cs" />
-    <Compile Include="Tasks\Aapt2.cs" />
-    <Compile Include="Tasks\Aapt2Compile.cs" />
-    <Compile Include="Tasks\Aapt2Link.cs" />
-    <Compile Include="Tasks\AndroidTask.cs" />
-    <Compile Include="Tasks\AndroidZipAlign.cs" />
-    <Compile Include="Tasks\BuildApk.cs" />
-    <Compile Include="Tasks\BuildBaseAppBundle.cs" />
-    <Compile Include="Tasks\BuildAppBundle.cs" />
-    <Compile Include="Tasks\BundleTool.cs" />
-    <Compile Include="Tasks\BuildApkSet.cs" />
-    <Compile Include="Tasks\BundleToolAdbTask.cs" />
-    <Compile Include="Tasks\InstallApkSet.cs" />
-    <Compile Include="Tasks\CilStrip.cs" />
-    <Compile Include="Tasks\ConvertDebuggingFiles.cs" />
-    <Compile Include="Tasks\CheckForInvalidResourceFileNames.cs" />
-    <Compile Include="Tasks\CollectNonEmptyDirectories.cs" />
-    <Compile Include="Tasks\CollectPdbFiles.cs" />
-    <Compile Include="Tasks\FilterAssemblies.cs" />
-    <Compile Include="Tasks\GenerateLibraryResources.cs" />
-    <Compile Include="Tasks\Generator.cs" />
-    <Compile Include="Tasks\JarToXml.cs" />
-    <Compile Include="Tasks\GetJavaPlatformJar.cs" />
-    <Compile Include="Tasks\GetFilesThatExist.cs" />
-    <Compile Include="Tasks\LinkAssembliesNoShrink.cs" />
-    <Compile Include="Tasks\ReadAndroidManifest.cs" />
-    <Compile Include="Tasks\RemoveRegisterAttribute.cs" />
-    <Compile Include="Tasks\GetMonoPlatformJar.cs" />
-    <Compile Include="Tasks\GeneratePackageManagerJava.cs" />
-    <Compile Include="Tasks\GenerateJavaStubs.cs" />
-    <Compile Include="Tasks\Javac.cs" />
-    <Compile Include="Tasks\LinkAssemblies.cs" />
-    <Compile Include="Tasks\CopyResource.cs" />
-    <Compile Include="Tasks\ResolveAndroidTooling.cs" />
-    <Compile Include="Tasks\ResolveAssemblies.cs" />
-    <Compile Include="Tasks\GetAppSettingsDirectory.cs" />
-    <Compile Include="Tasks\CopyIfChanged.cs" />
-    <Compile Include="Tasks\CreateResgenManifest.cs" />
-    <Compile Include="Tasks\CreateTemporaryDirectory.cs" />
-    <Compile Include="Tasks\GenerateResourceDesigner.cs" />
-    <Compile Include="Tasks\GetAndroidDefineConstants.cs" />
-    <Compile Include="Tasks\GetAndroidPackageName.cs" />
-    <Compile Include="Tasks\AndroidUpdateResDir.cs" />
-    <Compile Include="Tasks\AndroidSignPackage.cs" />
-    <Compile Include="Tasks\RemoveDirFixed.cs" />
-    <Compile Include="Tasks\ResolveJdkJvmPath.cs" />
-    <Compile Include="Tasks\ResolveSdksTask.cs" />
-    <Compile Include="Tasks\CompileToDalvik.cs" />
-    <Compile Include="Tasks\AndroidToolTask.cs" />
-    <Compile Include="Tasks\PrepareAbiItems.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
```

Comment:
Do we no longer need this metadata?  (I'm not sure what the metadata is for or who uses it, which makes it doubly difficult to reason about...)

Reply:
It seems that this metadata is not needed by short-form style projects, based on this comment https://github.com/dotnet/sdk/issues/1199#issuecomment-332052321